Multiplatform/Account/LoginQuickConnectView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  LoginQuickConnectView.swift
+//  AmpFin
+//
+//  Created by Daniel Cuevas on 9/27/24.
+//
+
+import SwiftUI
+import AmpFinKit
+
+internal struct LoginQuickConnectView: View {
+    let dismiss: (() -> Void)
+    @State var code: String?
+    @State var secret: String = ""
+    @State var success: Bool = false
+    @State var caughtError: Bool = false
+    
+    var body: some View {
+        
+        ScrollView {
+            VStack {
+                Spacer()
+                
+                if caughtError {
+                    VStack {
+                        Text("login.quickconnect.error")
+                            .font(.caption)
+                            .foregroundStyle(.red)
+                        Spacer()
+                    }
+                }
+                
+                Text(code ?? "??????")
+                    .font(.largeTitle)
+                    .padding(.bottom, 20)
+                
+                
+                ProgressView()
+                
+                Text(success ? "login.quickconnect.success" :(code != nil ? "login.quickconnect.waiting" : "login.quickconnect.preparing"))
+                    .font(.caption)
+                    .padding(.top, 4)
+                    .padding(.bottom, 20)
+                    .foregroundStyle(.secondary)
+                
+                
+                Button("login.quickconnect.copy", systemImage: "doc.on.doc") {
+                    UIPasteboard.general.string = code
+                }
+                .foregroundColor(.accentColor)
+                .disabled(code == nil)
+            }
+            .padding(.vertical, 40)
+        }
+        .interactiveDismissDisabled(success)
+        .onAppear() {
+            Task {
+                do {
+                    let (Code, Secret) = try await JellyfinClient.shared.initiateQuickConnect()
```

Comment:
You can leave out the `let` and just assign directly to `(code, secret)`. But this code will run on the MainActor (UI Thread) and consequently block it. But concurrency in Swift is quite complex, so I would just make this part of a Login view model, combined with the current login logic, but I am currently busy with ShelfPlayer.